src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    public abstract String onScreen();
```

Comment:
change method name, it's not clear what it does and it should be a verb. Plus, move it to a separate interface

Reply:
Changed